app/Views/ScBooking/Partial/AvailableTimes/_Trial.cshtml

```diff
@@ -0,0 +1,34 @@
+@model SCJ.Booking.MVC.ViewModels.ScCaseSearchViewModel
+
+@{
+    var availableDates = Json.Serialize(Model.AvailableDates).ToString();
+    @* Model.SelectedDate = Model.FirstAvailableDate; *@
+}
+
+<form method="post" id="availableTimesForm">
+
+  @Html.HiddenFor(m => m.HearingTypeId)
+  @Html.HiddenFor(m => m.EstimatedTrialLength)
+
+  <div id="VueTrialTimeSelect">
+    <trial-time-select-tabs>
+      <instant-book :dates="@availableDates" :trial-length="@Model.EstimatedTrialLength" slot="instantBook"></instant-book>
```

Comment:
It might be good to align this terminology with the SCSS API for consistency instead of introducing new terms.
e.g. **slot="instantBook"** could be changed to **formula="Regular"**

Reply:
Sounds good, I'll push an update to change the variable names in each branch, but I'll rename the actual vue files in the 142 branch only, to avoid potential rebasing chaos